fix setting migration issue

stefanbode opened the pull request:
## Description:
fixed issue with setting migration
**Related issue (if applicable):** fixes #<Tasmota issue number goes here>
#18326

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.7
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

stefanbode committed:
fix setting migration issue

arendst commented on tasmota/tasmota_xdrv_driver/xdrv_27_esp32_shutter.ino:
```diff
@@ -229,6 +229,7 @@ void ShutterSettingsDefault(void) {
     ShutterSettings.shutter_pwmrange[0][i] = pwm_servo_min;
     ShutterSettings.shutter_pwmrange[1][i] = pwm_servo_max;
   }
+  ShutterSettingsSave();
```
Are you sure this solves the issue? Doing it here it will overwrite the settings everytime with the default settings resetting the changes the user made.

stefanbode replied:
ok agree. let me double check.

stefanbode committed:
next try to fix migration issue

@arendst: I saw you made also some changes to the xdrv122. I copied this and it looks like setting the crc32 was the root of the issue. You removed it and I do the same now. Looks much better.

stefanbode commented:
@arendst: I saw you made also some changes to the xdrv122. I copied this and it looks like setting the crc32 in the load was the root of the issue. You removed it and I do the same now. Looks much better.
